src/panel_material_ui/base.py

```diff
@@ -171,7 +171,8 @@ class MaterialComponent(ReactComponent):
             "mui-color-input": "https://esm.sh/mui-color-input@6.0.0",
```

Comment:
Its not clear to me how to best select the versions. When I click on the versioned links I can see they refer to different versions of React and Material UI. Could that not give problems?